File: sample.js
class MatrixSum {
    constructor(r, c) {
        this.r = r;
        this.c = c;
        this.x = [];
        this.y = [];
        this.z = [];
    }
    input() {
        console.log("Enter elements of the 1st Matrix:");
        for (let i = 0; i < this.r; i++) {
            this.x[i] = [];
            for (let j = 0; j < this.c; j++) {
                this.x[i][j] = parseInt(prompt(Enter element [${i + 1}][${j + 1}] of matrix 1:), 10);
            }
        }
        console.log("Enter elements of the 2nd Matrix:");
        for (let i = 0; i < this.r; i++) {
            this.y[i] = [];
            for (let j = 0; j < this.c; j++) {
                this.y[i][j] = parseInt(prompt(Enter element [${i + 1}][${j + 1}] of matrix 2:), 10);
            }
        }
    }
    calculate() {
             for (let i = 0; i < this.r; i++) {
            this.z[i] = [];
            for (let j = 0; j < this.c; j++) {
                this.z[i][j] = this.x[i][j] + this.y[i][j];
            }
        }
    }
    display() {
         console.log("Output Matrix:");
        for (let i = 0; i < this.r; i++) {
            let row = "";
            for (let j = 0; j < this.c; j++) {
                row += this.z[i][j] + "\t";
            }
            console.log(row);
        }
    }
}
(function() {
    let r = parseInt(prompt("Enter the number of rows of the matrices:"), 10);
    let c = parseInt(prompt("Enter the number of columns of the matrices:"), 10);

    if (r > 0 && c > 0) {
        const obj = new MatrixSum(r, c);
        obj.input();
        obj.calculate();
        obj.display();
    } else {
        console.log("Error: Rows and columns should be greater than 0");
    }
})();

File: Sample.cs
using System;
class MatrixSum
{
    int r, c;
    int[,] x, y, z;
    public MatrixSum(int r1, int c1)
    {
        r = r1;
        c = c1;
        x = new int[r1, c1];
        y = new int[r1, c1];
        z = new int[r1, c1];
    }
    public void Input()
    {
        Console.WriteLine("Enter elements of the 1st Matrix:");
        for (int i = 0; i < r; i++)
        {
            for (int j = 0; j < c; j++)
            {
                x[i, j] = int.Parse(Console.ReadLine());
            }
        }

        Console.WriteLine("Enter elements of the 2nd Matrix:");
        for (int i = 0; i < r; i++)
        {
            for (int j = 0; j < c; j++)
            {
                y[i, j] = int.Parse(Console.ReadLine());
            }
        }
    }
    public void Calculate()
    {
        for (int i = 0; i < r; i++)
        {
            for (int j = 0; j < c; j++)
            {
                z[i, j] = x[i, j] + y[i, j];
            }
        }
    }
    public void Display()
    {
        Console.WriteLine("Output Matrix:");
        for (int i = 0; i < r; i++)
        {
            for (int j = 0; j < c; j++)
            {
                Console.Write(z[i, j] + "\t");
            }
            Console.WriteLine();
        }
    }
    static void Main(string[] args)
    {
        Console.WriteLine("Enter the number of rows and columns of the matrices:");
        int r = int.Parse(Console.ReadLine());
        int c = int.Parse(Console.ReadLine());

        if (r > 0 && c > 0)
        {
            MatrixSum obj = new MatrixSum(r, c);
            obj.Input();
            obj.Calculate();
            obj.Display();
        }
        else
        {
            Console.WriteLine("Error: Rows and columns should be greater than 0");
        }
    }
}
